commands/files/serialfile.go

```diff
@@ -84,10 +84,25 @@ func (f *serialFile) NextFile() (File, error) {
 	// open the next file
 	fileName := fp.Join(f.name, stat.Name())
 	filePath := fp.Join(f.path, stat.Name())
+	st, err := os.Lstat(filePath)
```

Comment:
In that case, it wouldnt make sense not to follow the symlink. What are you going to do with a single symlink inside ipfs? A symlink out of context is fairly useless, and potentially dangerous.